python/mp/effects/effect.py

```diff
@@ -80,7 +80,7 @@ def generate_osc_path(self, fn_name):
         Centralize the dispatcher path name creation
         """
 
-        return "/effect/{}/{}/{}".format(self.rosary.name,
+        return "/{}/effect/{}/{}".format(self.rosary.name,
```

Comment:
Thought this would make the paths more intuitive. Don't imagine anyone cares terribly much. Let me know.